src/controllers/write/bugs.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const bugController = module.exports;
+const bugCreate = require('../../bugs');
+
+bugController.submitData = async function (req, res) {
+    bugCreate.post(req.body);
+    console.log(req.body);
```

Comment:
Lets remove some of the logging here as well

Reply:
Fixed